src/features/user/MFV2/ForestProgress/TreeTargetBar.module.scss

```diff
@@ -0,0 +1,247 @@
+  margin-bottom: 7px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.barMainContainer {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  font-size: 12px;
+  font-weight: 700;
+}
+
+.barContainer {
+  height: 6px;
+  width: 100%;
+  border-radius: 5px;
+  display: flex;
+  background-color: #fff;
+}
+
+.communityReceived {
+  font-weight: 400;
+  font-size: 10px;
+}
+
+.statisticsMainContainer {
+  display: flex;
+  width: 100%;
+  gap: 14px;
+  justify-content: center;
+  align-items: center;
+  line-height: 9px;
+}
+
+.statisticsContainer {
+  width: 100%;
+  height: 52px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  .stat {
+    font-size: 12px;
+    font-weight: 700;
+  }
+}
+
+.editTargetContainer {
+  display: flex;
+  gap: 4px;
+  cursor: pointer;
+}
+
+.editTargetLabel {
+  font-size: 10px;
+  font-weight: 700;
+  line-height: 8px;
+}
+
+//tree target modal
+
+.crossIconContainer {
+  position: absolute;
+  right: 18px;
+  top: 18px;
+  cursor: pointer;
+}
+
+.targetModalMainContainer {
+  position: relative;
+  left: 50%;
+  top: 50%;
+  width: 398px;
+  height: 524px;
+  background: #fff;
+  transform: translate(-50%, -50%);
+  border-radius: 16px;
+  padding: 18px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.saveButton {
+  width: 100%;
+  height: 45px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: linear-gradient(0deg, #008f5c 0%, #008f5c 100%);
+  border-radius: 12px;
+  color: #fff;
+  font-size: 16px;
+  font-weight: 700;
+  cursor: pointer;
+}
+
+.targetModalSubConatiner {
+  display: flex;
+  gap: 14px;
+  flex-direction: column;
+}
+
+.deActivateTargetModal {
+  .targetInputIconContainer {
+    background: #828282;
+  }
+}
+
+.deActivateTargetModal,
+.targetFormInputContainer {
+  width: 100%;
+  height: 120px;
+  border-radius: 8px;
+  gap: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 16px;
+}
+
+.switchContainer {
+  display: flex;
+  justify-content: space-between;
+}
+
+.targetInputIconMainContainer {
+  display: flex;
+  align-items: center;
+  gap: 6px;
+  .label {
+    display: flex;
+    align-items: center;
+    font-size: 12px;
+    font-weight: 700;
+  }
+}
+
+.targetInputIconContainer {
+  width: 24px;
+  height: 24px;
+  border-radius: 3.1px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.setTargetLabel {
+  display: flex;
+  justify-content: center;
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.editTargetButtonContainer {
+  display: flex;
+  width: 100%;
+  justify-content: flex-end;
+  position: absolute;
+  top: 10px;
+  right: 10px;
+}
+
+.personalPercentageBar {
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+}
+
+.giftPercentageBar {
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+}
+
+//target initialize  copntainer style
+
+.EmptyProgressContainer {
```

Comment:
Please name classes consistently i.e. .emptyProgressContainer as done for the other classes.